kora.toml

```diff
@@ -7,12 +7,40 @@ max_signatures = 10
 allowed_programs = [
     "11111111111111111111111111111111",  # System Program
     "TokenkegQfeZyiNwAJbNbGKPFXCWuBvf9Ss623VQ5DA",  # Token Program
-    "ATokenGPvbdGVxr1b2hvZbsiqW5xWH25efTNsLJA8knL",
+    "ATokenGPvbdGVxr1b2hvZbsiqW5xWH25efTNsLJA8knL", # Ata Program
 ]
 allowed_tokens = [
     "4zMMC9srt5Ri5X14GAgXhaHii3GnPAEERYPJgZJDncDU",  # USDC devnet
 ]
 allowed_spl_paid_tokens = [
     "4zMMC9srt5Ri5X14GAgXhaHii3GnPAEERYPJgZJDncDU",  # USDC devnet
-] 
-disallowed_accounts = []
\ No newline at end of file
+]
+disallowed_accounts = []
+# if there is an "allowed_programs" listed above and not in the allowed_instructions, then we will allow all instructions for that program
+allowed_instructions = [
+    { program = "TokenkegQfeZyiNwAJbNbGKPFXCWuBvf9Ss623VQ5DA", instructions = ["[3,...]"] }, # when there's no IDL available
+    { program = "11111111111111111111111111111111", instructions = ["transfer"] }, # when there's an IDL available
+    { program = "ATokenGPvbdGVxr1b2hvZbsiqW5xWH25efTNsLJA8knL", instructions = ["*"] }, # Ata Program
+]
+# 1. how do I convert from string to u8 discriminators?
```

Comment:
I'll clean all this up in a future PR as well as add sane example allowed instructions.

Side comment:
I'm considering enabling a couple smart defaults like "transfers" or "defi" that would automatically build an allowlist for good defaults (USDC/USDT etc transfers, well known defi protocols etc) What do you think? worth it? (can be a follow up PR)

Reply:
Yea this would be a huge value-add.

Grab some Raydium, Jupiter, and Orca discriminators